ee/tasks/usage_report.py

```diff
@@ -413,6 +415,18 @@ def send_org_usage_report(
     except Exception as err:
         logger.error("Usage report for organization %s failed!", organization_id)
         if not dry_run:
+            # If we except we still want to capture the minimum amount of info for the Billing Service
+            minimal_report_dict = {
+                "organization_id": organization_id,
+                "date": period_start.strftime("%Y-%m-%d"),
+                "event_count_in_period": get_agg_event_count_for_teams_and_period(team_ids, period_start, period_end),
+                "recording_count_in_period": get_agg_recording_count_for_teams_and_period(
+                    team_ids, period_start, period_end
+                ),
+            }
+            billing_service_token = build_billing_token(license, organization_id) if license else None
+            send_report_to_billing_service(minimal_report_dict, billing_service_token)
```

Comment:
Hmm how do we know we're not excepting anywhere in this block? Anyways might be worth to get this in and see if it ends up fixing things. Can't break what's already broken 🤷

Reply:
Yeah we won't know but we should find out from the overall task excepting (hard fail feels good for this case where we can't even talk to the billing service

Reply:
In that case, do you think we should move up the `capture_exception` and `capture_event` above this block to ensure the exceptions get captured?